src/controllers/TalksController.php

```diff
@@ -647,10 +646,10 @@ public function setSpeakerForTalk(Request $request, PDO $db)
 
     private function getLinkUserDataFromRequest(Request $request)
     {
-        $talk = [];
-        $talk['display_name'] = trim($request->getParameter('display_name', ''));
-        $talk['username'] = trim($request->getParameter('username', ''));
-        return $talk;
+        return [
+            'display_name' => trim($request->getParameter('display_name', '')),
+            'username' => trim($request->getParameter('username', '')),
```

Comment:
Shouldn't the ```=>``` be aligned under each other?

Reply:
That would look good. 43bb031